content/en/docs/v3/guides/infra/tls_dns.md

```diff
@@ -0,0 +1,140 @@
+---
+title: TLS and DNS
+linktitle: TLS and DNS
+description: Automated TLS and DNS
+weight: 35
+---
+
+This section will describe how to enable automated TLS and DNS for your Jenkins X installation.
+
+To achive this we will use a couple of open source projects to help enable automated DNS for applications you 
+
+For this guide we are going to assume you own a domain called `foo.io` which is managed by Google Cloud DNS, if it is not see [configure cloud dns to manage a domain](/docs/v3/guides/infra/google_cloud_dns).  
+
+Jenkins X services will have URLs like `https://hook-jx.bar.foo.io`.  The jx-requirement.yaml `namespaceSubDomain:` of `-jx` refers to the Kubernetes namespace the service is running in, this helps avoid clashes of the same application running in different namespaces in the same cluster.
+
+To start with we are focussed on GCP but will expand to other cloud providers.
+
+# Google Cloud Platform
+
+## Prerequisits
+
+- cluster created using Jenkins X [GCP Terraform getting started](/docs/v3/getting-started/gke/)
+- own a domain, we will use [Google Domains](https://domains.google.com/registrar/) in this guide but any provider will work
+- latest Jenkins X [upgrade](/docs/v3/guides/upgrade)
+
+### Cloud Infrastructure
+First we will configure the cloud infrastructure requirements:
+
+- a GCP Service Account with the `dns.admin` role, see [here](https://cloud.google.com/iam/docs/understanding-roles#dns-roles) for more information
+- a managed cloud dns zone, see [here](https://cloud.google.com/dns/docs/zones) for more information
+
+To satisfy these requirements go to your infrastructure repository (contains Terraform main.tf) and add to your `values.auto.tfvars` the following:
+
+```values.tf
+parent_domain = "foo.io"
+```
+
+Most people prefer to use a subdomain for a specific installation rather than purchasing one domain per cluster.  For example in a multi cluster setup you will probably want the same parent domain but each cluster using a differnt subdomain like development.foo.io, staging.foo.io and foo.io.
```

Comment:
```suggestion
Most people prefer to use a subdomain for a specific installation rather than purchasing one domain per cluster.  For example in a multi cluster setup you will probably want the same parent domain but each cluster using a different subdomain like development.foo.io, staging.foo.io and foo.io.
```

Reply:
`foo.io` is a parent domain not a subdomain